robottelo/libvirt_discovery.py

```diff
@@ -56,7 +56,13 @@ def __init__(
         else:
             self.image_dir = image_dir
         if mac is None:
-            self.mac = gen_mac(multicast=False, locally=True)
+            # fe:* MAC range is considered reserved in libvirt
+            while True:
+                mac = gen_mac(multicast=False, locally=True)
+                if not mac.startswith(u'fe'):
```

Comment:
can we use `multicast=True` ?

```
In [3]: while True:
   ...:     mac = gen_mac(multicast=True, locally=True)
   ...:     if mac.startswith('fe'):
   ...:         break
   # never breaks, because faux_factory ensure first octet to be started with 1
```

https://github.com/omaciel/fauxfactory/blob/master/fauxfactory/__init__.py#L630